src/components/GrainResults.js

```diff
@@ -48,6 +53,14 @@ const GrainResults = () => {
     );
   });
 
+  const handleOpenModal = () => {
```

Comment:
I think having the map modal here as well is a good idea. Thanks for implementing it -- and taking the time to have folks look at the app and give feedback!